mongodbatlas/data_source_mongodbatlas_cloud_provider_access.go

```diff
@@ -0,0 +1,151 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	matlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const (
+	errorGetRead = "error reading cloud provider access %s"
+)
+
+func dataSourceMongoDBAtlasCloudProviderAccessList() *schema.Resource {
```

Comment:
is it? @leofigy

Reply:
It's weird, I did a PR but it was to grab your changes into my branch https://github.com/mongodb/terraform-provider-mongodbatlas/pull/353, because I needed your changes because you changed the client. I did a commit right now to remove the provider access files